### ISSUE TITLE
RSS/Atom feed for blog

### ISSUE THREAD

**Issue by haroldcarr** [unisonweb/website#27]
Does your blog have an RSS/Atom feed?

My reader, [https://bazqux.com/](https://bazqux.com/) (written in Haskell and Ur/Web, BTW) cannot find it.

**Comment by gampleman** [comment]
Seems like this has been fixed finally.

**Comment by rlmark** [comment]
Yes! Closing out some old issues in this repo. For future devs, the Atom feed is at https://www.unison-lang.org/feed.xml.
